.evergreen/config.yml

```diff
@@ -32,6 +32,11 @@ functions:
       type: system
       params:
         directory: src/go.mongodb.org/mongo-driver
+    - command: shell.exec
+      params:
+        working_dir: "src/go.mongodb.org/mongo-driver"
+        script: |
+          git submodule update --init
```

Comment:
Optional: Consider using the `init-submodule` task instead.

Reply:
This solution doesn't work for windows for some reason 🤷

Reply:
That's weird. Oh well, 🤷